configure

```diff
@@ -1,4 +1,14 @@
-#!/usr/bin/env python
+#!/bin/sh
+
+# Locate python2 interpreter and re-execute the script.  Note that the
+# mix of single and double quotes is intentional, as is the fact that
+# the ] goes on a new line.
+_=[ 'exec' '/bin/sh' '-c' '''
```

Comment:
Is changing `$_` definitely okay in all shells? I know it's normally not user-modifiable.